src/main/java/com/google/sps/servlets/SurveyServlet.java

```diff
@@ -15,8 +15,11 @@
 import com.google.cloud.datastore.Value;
 import com.google.common.collect.ImmutableMap;
 import com.google.cloud.Timestamp;
+import com.google.gson.*;
```

Comment:
Do you need everything under gson?

Reply:
Nope! Switched to only importing Gson in e8e6bb25efc414562cdcc6a779c82054587487fb